### PR TITLE
Fixconfigcrash

### PR DESCRIPTION
**Pull request by ggggggggg**
fixes a crash when EdgeMulti is saved as true but the EMT state is not saved

also fixes some EMT crashes

### CODE REVIEW

File: abaco_test.go [hunk 1]
```diff
@@ -78,7 +78,7 @@ func TestGeneratePackets(t *testing.T) {
 	}
 }
 
-func _skip_TestAbacoRing(t *testing.T) {
+func TestAbacoRing(t *testing.T) {
```
Comment (joefowler) [inline abaco_test.go:81]: I don't understand why these were skipped at all. Did you commit to Master without me noticing? In my version of master they aren't skipped.

**Review by joefowler — APPROVED** [review]
If this fixes #271, then great. I'm pushing a change to update the Dastard version number to 0.2.13 and add a comment to `VERSION.md`.

File: abaco_test.go (hunk 1 above)
Comment (ggggggggg) [reply abaco_test.go:81]: I must have. They crash a lot, so when I was running tests in the debugger, I had to skip them to actually catch the crashes I was interested in in the EMT code.

Comment (joefowler) [reply abaco_test.go:81]: Bad Galen! Good for restoring them.

Comment (ggggggggg) [reply abaco_test.go:81]: No, bad tests!!! Brilliant skipping to make progress!

**Pull request merged**